test_models.py

```
@@ -0,0 +1,361 @@
+        
+        return audit
+
+
+async def test_configuration_models(device: Device):
+    """Test ConfigurationSnapshot and ConfigurationChangeEvent models."""
+    print("🧪 Testing Configuration models...")
+    
+    async with get_async_session() as session:
+        # Create configuration snapshot
+        snapshot = ConfigurationSnapshot(
+            time=datetime.now(timezone.utc),
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)